queries/nim/highlights.scm

```diff
@@ -0,0 +1,797 @@
+; SPDX-FileCopyrightText: 2023 Leorize <leorize+oss@disroot.org>, aMOPel <>
+; SPDX-License-Identifier: MPL-2.0
+; SPDX-License-Identifier: Apache-2.0
+
+; =============================================================================
+; catch all rules
+
+((identifier) @variable (#set! "priority" 95))
+
+; =============================================================================
+; @comment               ; line and block comments
+
+[
+  (comment)
+  (block_comment)
+] @comment
+
+; =============================================================================
+; @comment.documentation ; comments documenting code
+
+[
+  (documentation_comment)
+  (block_documentation_comment)
+] @comment.documentation
+
+; =============================================================================
+; @none                  ; completely disable the highlight
+; unused
+
+; =============================================================================
+; @define                ; preprocessor definition directives
+; unused
+
+; =============================================================================
+; @operator              ; symbolic operators (e.g. `+` / `*`)
+
+(operator) @operator
+
+; =============================================================================
+; @punctuation.delimiter ; delimiters (e.g. `;` / `.` / `,`)
+
+[ "." ";" "," ":" "=" ] @punctuation.delimiter
+
+; needs to be after @punctuation.delimiter
+(assignment "=" @operator)
```

Comment:
Move the `(operator)` capture below instead?

Reply:
sure